### DISCUSSION TITLE
Security Issue Insight Check (with you, my peers)

### DISCUSSION THREAD

**Discussion by Ohhhhimmm** [community/community#73675] [Programming Help]
Attached I have a screenshot of my Chromebook and you can see new rules that were set up in my uBlock extension in my Chrome broswer. The thing that's scariest is that these rules were put into place WHILE I was trying to get my settings int he right place for me to be able to browse semi-safely... Can anyone make heads or tails of this code and give me a heads up as to what measures I should take for their reversal/removal?

Please and thank you. 

T

![Screenshot 2023-11-06 14 15 12](https://github.com/community/community/assets/149048211/cec39307-44cb-4e40-a81a-2cf5f708d309)

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @Ohhhhimmm , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
